tests/integration/test_interaction.py

```diff
@@ -424,6 +472,12 @@ def test_item_no_feedback_on_bad_move(self, action_key):
         )
 
     @data(None, Keys.RETURN, Keys.SPACE, Keys.CONTROL+'m', Keys.COMMAND+'m')
+    def test_move_items_between_zones(self, action_key):
+        self.parameterized_move_items_between_zones(
```

Comment:
Is there test coverage that items can't be moved once dropped for assessment_mode=False?